packages/datadog_flutter_plugin/lib/src/tracing/tracing_headers.dart

```diff
@@ -169,8 +169,8 @@ Map<String, String> getTracingHeaders(
     case TracingHeaderType.b3:
       if (context.sampled) {
         final headerValue = [
-          context.spanId.asString(TraceIdRepresentation.hex32Chars),
-          context.traceId.asString(TraceIdRepresentation.hex16Chars),
+          context.traceId.asString(TraceIdRepresentation.hex32Chars),
+          context.spanId.asString(TraceIdRepresentation.hex16Chars),
```

Comment:
just curious: what is the reason of having 32 vs 16 chars?

Reply:
64 vs 128 bits. We only generate 63-bit ids but both b3 and traceheader trace Ids should be 128 bits (spans are still 64). We pad the beginning with 0's to bring it up to 128 bits.